packages/progress-indicator/src/LionProgressIndicator.js

```diff
@@ -78,12 +130,64 @@ export class LionProgressIndicator extends LocalizeMixin(LitElement) {
 
   connectedCallback() {
     super.connectedCallback();
-    this.setAttribute('role', 'status');
-    this.setAttribute('aria-live', 'polite');
+    this.setAttribute('role', 'progressbar');
+  }
+
+  /**
+   * Update aria labels on state change.
+   * @param {import('@lion/core').PropertyValues } changedProperties
+   */
+  updated(changedProperties) {
+    super.updated(changedProperties);
+
+    if (this.indeterminate) {
+      if (changedProperties.has('_ariaLabel') || changedProperties.has('_ariaLabelledby')) {
+        this._setDefaultLabel();
+      }
+    } else {
+      if (changedProperties.has('value')) {
+        if (this.value < this.min) {
+          this.value = this.min;
+          this.setAttribute('aria-valuenow', this.min.toString());
+        } else if (this.value > this.max) {
+          this.value = this.max;
+          this.setAttribute('aria-valuenow', this.max.toString());
+        } else {
+          this.setAttribute('aria-valuenow', this.value.toString());
+        }
+        if (this.__hasDefaultLabelSet === true) {
+          this.removeAttribute('aria-label');
```

Comment:
```suggestion
          this.removeAttribute('aria-label');
          this.__hasDefaultLabelSet = false;
```

And then a test that makes sure that